nginx/amiable.sh

```diff
@@ -5,7 +5,13 @@
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 nginxHome=`nginx -V 2>&1 | grep "configure arguments:" | sed 's/[^*]*conf-path=\([^ ]*\)\/nginx\.conf.*/\1/g'`
 
-sudo ln -fs $DIR/amiable.conf $nginxHome/sites-enabled/amiable.conf
+if [ -d "${nginxHome}/sites-enabled" ]; then
+  confDir=sites-enabled
+else
+  confDir=servers
+fi
```

Comment:
Unfortunately it will still not work as the directory will exist, it is just that the in the main configuration file, `servers` is  included instead of `sites-enabled`. I think It may be easier to tell people to change that line if there are using `OS x`. I know `https://github.com/guardian/dev-nginx` has the same issue.